qdrant-landing/content/documentation/quick-start.md

```diff
@@ -54,151 +35,67 @@ client = QdrantClient(":memory:")
+```
 
-* [Python](https://github.com/qdrant/qdrant-client) - `pip install qdrant-client`
-* [Rust](https://github.com/qdrant/rust-client) - `cargo add qdrant-client`
-* [Go](https://github.com/qdrant/go-client) - `go get github.com/qdrant/go-client`
-* [Javascript](https://github.com/qdrant/qdrant-js) - `npm install @qdrant/js-client-rest`
+<aside role="status">Qdrant has no encryption or authentication by default and new instances are open to everyone. Please read <a href="https://qdrant.tech/documentation/security/">Security</a> carefully for details on how to secure your instance.</aside>
 
-If you are using a language that is not listed here, you can use the REST API directly or generate a client for your language 
-using [OpenAPI](https://github.com/qdrant/qdrant/blob/master/docs/redoc/master/openapi.json)
-or [protobuf](https://github.com/qdrant/qdrant/tree/master/lib/api/src/grpc/proto) definitions. 
+# Running vector search queries
 
-### Create collection
+In this short example, you will create a Qdrant collection, load data into it and run a basic search query. 
 
-First - let's create a collection with dot-production metric.
+![Qdrant Quickstart](/docs/quickstart.png)
 
-```bash
-curl -X PUT 'http://localhost:6333/collections/test_collection' \
-    -H 'Content-Type: application/json' \
-    --data-raw '{
-        "vectors": {
-            "size": 4,
-            "distance": "Dot"
-        }
-    }'
-```
+## Step 1. Create a collection
+
+You will be storing all of your vector data in a Qdrant collection. Let's call it `test_collection`. This collection will be using a doc production metric. 
 
 ```python
-from qdrant_client import QdrantClient
+
 from qdrant_client.http.models import Distance, VectorParams
 
-client = QdrantClient("localhost", port=6333)
 client.recreate_collection(
     collection_name="test_collection",
     vectors_config=VectorParams(size=4, distance=Distance.DOT),
 )
 ```
 
-Expected response:
-
-```json
-{
-    "result": true,
-    "status": "ok",
-    "time": 0.031095451
-}
-```
-
-We can ensure that collection was created:
-
-```bash
-curl 'http://localhost:6333/collections/test_collection'
-```
-
-```python
-collection_info = client.get_collection(collection_name="test_collection")
-```
-
-Expected response:
-
-```json
-{
-  "result": {
-    "status": "green",
-    "vectors_count": 0,
-    "segments_count": 5,
-    "disk_data_size": 0,
-    "ram_data_size": 0,
-    "config": {
-      "params": {
-        "vectors": {
-          "size": 4,
-          "distance": "Dot"
-        }
-      },
-      "hnsw_config": { ... },
-      "optimizer_config": { ... },
-      "wal_config": { ... }
-    }
-  },
-  "status": "ok",
-  "time": 2.1199e-05
-}
-```
+**Response:**
 
 ```python
-from qdrant_client.http.models import CollectionStatus
-
-assert collection_info.status == CollectionStatus.GREEN
-assert collection_info.vectors_count == 0
+True
 ```
 
-### Add points
+## Step 2: Add vectors
```

Comment:
I don't think it is a good idea to have differing payloads for each of the vectors. In the first vector, the city payload item is just a single entity while in the next 3 it is an array. 
And then the next two vectors do something else entirely and as a reader I am unsure what this means and why it is like this. 

For me trying to understand this code is harder than it need be just for a quick start. I do think it is important to show the flexibility of the payload but not here.

Reply:
How do you want the vectors to look? Can you write out the payloads?

Reply:
```
          {"id": 1, "vector": [0.05, 0.61, 0.76, 0.74], "payload": {"city": "Berlin" }},
          {"id": 2, "vector": [0.19, 0.81, 0.75, -0.11], "payload": {"city": "London" }},
          {"id": 3, "vector": [0.36, -0.55, 0.47, 0.94], "payload": {"city": "Moscow" }},
          {"id": 4, "vector": [0.18, 0.01, 0.85, 0.80], "payload": {"city": "New York" }}, 
          {"id": 5, "vector": [0.24, 0.18, 0.22, 0.44], "payload": {"city": "Beijing" }}, 
          {"id": 6, "vector": [0.35, 0.08, 0.11, 0.44], "payload": {"city": "Mumbai" }}, 
```

Reply:
Sorry this makes you redo the sample queries and such below.